src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.Client.fs

```diff
@@ -185,6 +194,8 @@ type Client<'TAttribute, 'TContext when 'TAttribute :> AnalyzerAttribute and 'TC
                     then
                         true
                     else
+                        System.Threading.Interlocked.Increment skippedAssemblies |> ignore
```

Comment:
(This is an idiom I strongly prefer, because it's robust against people choosing in the future to parallelise the `Array.filter`. Of course, I could rewrite this as a fold instead if mutability is considered harmful.)